e2e/testcases/helm_sync_test.go

```diff
@@ -396,24 +397,41 @@ image:
 // are set. Namespace-scoped resources with their namespace field set should be deployed to their set namespace; namespace-scoped
 // resources with their namespace field not set should be deployed to the default namespace.
 // This test will work only with following pre-requisites:
-// Google service account `e2e-test-ar-reader@${GCP_PROJECT}.iam.gserviceaccount.com` is created with `roles/artifactregistry.reader` for accessing images in Artifact Registry.
+// Google service account `e2e-test-ar-reader@${GCP_PROJECT}.iam.gserviceaccount.com` is created.
 func TestHelmDefaultNamespace(t *testing.T) {
```

Comment:
This conflicts with several of my changes in https://github.com/GoogleContainerTools/kpt-config-sync/pull/1113. I would have preferred to get those refactoring changes in first, but if necessary I can rebase on this change

Reply:
This PR needs to be cherry-picked on to the release branch, but #1113 probably doesn't have to.
To minimize changes in a patch release, I would rather #1113 to be rebased any merged after.

Reply:
Yeah, that makes sense. Unfortunately I'll have to manually revalidate those tests for each of the gke/provider permutations after rebasing 😢

One of the costs for cherry picking a feature

Reply:
Agreed and sorry for the extra costs.

If not cherry picking the feature in a patch release, it will block other services :disappointed: